.env.development

```diff
@@ -1 +1,2 @@
-API_BASE_URL=http://localhost:23500
+NEXT_PUBLIC_API_BASE_URL=http://localhost:23500
+NEXT_PUBLIC_FETCH_MOCK_DATA=true
```

Comment:
Now that I come to think about it, `NEXT_PUBLIC_FETCH_MOCK_DATA`  should not be updated to the repository, as this is a value that each developer should be free to switch on and off during development. Will move it to some local environment file in the following PR.